client/routes/domain/workflow-archival/workflow-archival-service.js

```diff
@@ -0,0 +1,33 @@
+import { mapArchivedWorkflowResponse } from './helpers';
+import { http } from '~helpers';
+
+export default ({ domain }) => {
+  const requests = {
+    fetchArchivalRecords: null,
+  };
+
+  return {
+    fetchArchivalRecords: async query => {
+      if (requests.fetchArchivalRecords) {
+        requests.fetchArchivalRecords.abort();
+      }
+
+      const controller = new AbortController();
```

Comment:
Since AbortController is an experimental JS feature, maybe add a few tests to validate at least the API hasn't changed?

Reply:
can do

Reply:
I'll add a unit test around the service so that if it fails it will fail the unit test.